scripts/hotrod-integration-test.sh

```diff
@@ -30,4 +66,18 @@ if [[ $body != *"Rides On Demand"* ]]; then
 fi
 docker rm -f "$CID"
 
-bash scripts/build-upload-a-docker-image.sh -c example-hotrod -d examples/hotrod -p "${platforms}"
+# build and upload image to dockerhub/quay.io
+bash scripts/build-upload-a-docker-image.sh ${LOCAL_FLAG} -c "${BINARY}" -d examples/hotrod -p "${platforms}"
+
+# build debug image if requested
+if [[ "${add_debugger}" == "Y" ]]; then
+  make build-examples GOOS=linux GOARCH="${GOARCH}" DEBUG_BINARY=1
+  repo="${REPO}-debug"
+
+  # build locally for integration test (the -l switch)
+  bash scripts/build-upload-a-docker-image.sh -l -c "${BINARY}-debug" -d examples/hotrod -t debug
+  run_integration_test "localhost:5000/$repo"
```

Comment:
what is `run_integration_test`?

Reply:
Written the corresponding integration test for local